cmis/views/cmis_backend.xml

```diff
@@ -1,75 +1,74 @@
+                        <group colspan="4" col="4">
+                            <newline />
+                            <field name="initial_directory_write" />
+                            <button name="check_directory_of_write" type="object" class="oe_highlight" string="Test Directory" />
                         </group>
-                    </sheet>
-                </form>
-            </field>
-        </record>
+                    </group>
+                </sheet>
+            </form>
+        </field>
+    </record>
 
-        <record model="ir.ui.view" id="cmis_backend_search_view">
-            <field name="name">cmis.backend.search (in cmis)</field>
-            <field name="model">cmis.backend</field>
-            <field name="arch" type="xml">
-                <search string="CMIS Backend">
-                    <field name="name"/>
-                    <field name="location"/>
-                </search>
-            </field>
-        </record>
+    <record model="ir.ui.view" id="cmis_backend_search_view">
+        <field name="name">cmis.backend.search (in cmis)</field>
+        <field name="model">cmis.backend</field>
+        <field name="arch" type="xml">
+            <search string="CMIS Backend">
+                <field name="name"/>
+                <field name="location"/>
+            </search>
+        </field>
+    </record>
 
-        <record model="ir.ui.view" id="cmis_backend_tree_view">
-            <field name="name">cmis.backend.tree (in cmis)</field>
-            <field name="model">cmis.backend</field>
-            <field name="arch" type="xml">
-                <tree string="Claim Categories">
-                    <field name="name" widget="handle" />
-                    <field name="location"/>
-                </tree>
-            </field>
-        </record>
+    <record model="ir.ui.view" id="cmis_backend_tree_view">
+        <field name="name">cmis.backend.tree (in cmis)</field>
+        <field name="model">cmis.backend</field>
+        <field name="arch" type="xml">
+            <tree>
+                <field name="name" widget="handle"/>
+                <field name="location"/>
+            </tree>
+        </field>
+    </record>
 
-        <record model="ir.actions.act_window" id="cmis_backend_act_window">
-            <field name="name">Cmis Backend</field> <!-- TODO -->
-            <field name="res_model">cmis.backend</field>
-            <field name="view_mode">tree,form</field>
-            <field name="domain">[]</field>
-            <field name="context">{}</field>
-        </record>
+    <record model="ir.actions.act_window" id="cmis_backend_act_window">
+        <field name="name">Cmis Backend</field>
+        <field name="res_model">cmis.backend</field>
+        <field name="view_mode">tree,form</field>
+        <field name="domain">[]</field>
+        <field name="context">{}</field>
+    </record>
 
+    <menuitem id="cmis_root_menu"
```

Comment:
Hi @max3903 , just a real detail... is there a special reason to re-introduce here this synonym form of an `ir.ui.menu` record declaration ?   What's exactly the added value ?   The classic `record` form has maybe a better longevity!

Reply:
@Olivier-LAURENT If you use `record`, it doesn't set the parent_left and parent_right and the menu is never displayed.